README.md

```diff
@@ -52,3 +52,139 @@ For a basic setup allowing you to build the API and run tests:
 ## API Reference
 
 See the [Vega-Lite JavaScript API Reference](https://vega.github.io/vega-lite-api/api).
+
+## Usage
+
+### vega-lite API For Observable Notebooks
+
+Just import it like this:
+
+~~~js
+import {vl} from '@vega/vega-lite-api'
+~~~
+
+### vega-lite API for Browsers
+
+To use the vega-lite API on a browser, you need to include all the dependencies, set the default configuration and finally register it. Here is some starting code you can build from
+
+~~~html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+  </head>
+  <body>
+    <div id="chart"></div>
+
+    <script src="https://cdn.jsdelivr.net/npm/vega"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite-api"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-tooltip"></script>
+
+    <script>
+      const options = {
+        config: {
+          // vega-lite default configuration
+        },
+        init: (view) => {
+          // initialize tooltip handler
+          view.tooltip(new vegaTooltip.Handler().call);
+          // enable horizontal scrolling for large plots
+          if (view.container()) view.container().style["overflow-x"] = "auto";
```

Comment:
Why don't we use Vega-Embed, which already comes with tooltips?

Reply:
Sounds good to me if it simplifies the code 👍